pulp_container/tests/functional/conftest.py

```diff
@@ -297,3 +301,26 @@ def container_signature_api(container_client):
 def token_server_url(cli_client):
     """The URL of the token server."""
     return get_pulp_setting(cli_client, "TOKEN_SERVER")
+
+
+@pytest.fixture
+def container_repo(container_repository_api, gen_object_with_cleanup):
+    return gen_object_with_cleanup(
+        container_repository_api, ContainerContainerRepository(**gen_repo())
+    )
+
+
+@pytest.fixture
+def container_remote(container_remote_api, gen_object_with_cleanup):
+    return gen_object_with_cleanup(container_remote_api, gen_container_remote())
+
+
+@pytest.fixture
+def sync(container_repository_api, monitor_task):
+    def _sync(remote, repo):
```

Comment:
What if we make this account for repositories with preconfigured remotes?

```suggestion
    def _sync(repo, remote=None):
```

Reply:
Probably we will need to check on the repo to then call the `sync`.

Reply:
What do we need to check? Sync will fail if a remote is neither on the repository, nor specified in the call.